packages/core/src/components/spinner/spinner.tsx

```diff
@@ -77,6 +79,15 @@ export class Spinner extends React.PureComponent<ISpinnerProps, {}> {
         );
     }
 
+    protected validateProps({ className = "" }: ISpinnerProps) {
+        if (className.indexOf(Classes.SMALL) >= 0) {
+            console.warn(SPINNER_WARN_CLASSES_SMALL);
```

Comment:
Shouldn't we just add support for the classes?

Reply:
moved to #2667